tests/PoeditorTest.php

```diff
@@ -81,6 +82,16 @@ public function it_requests_export_url_and_downloads_it_content()
         $this->assertEquals(parse_url($url, PHP_URL_PATH), $downloadExportRequest->getUri()->getPath());
     }
 
+    /** @test */
+    public function it_throws_an_error_if_config_values_are_empty()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        config()->set('poeditor-sync.api_key', '');
+        config()->set('poeditor-sync.project_id', '');
```

Comment:
you should check one empty config value at a time, so you are sure both config values are validated.